code/game/jobs/job/security.dm

```diff
@@ -230,3 +231,54 @@
 	name = "Marshal Officer"
 	icon_state = "player-blue"
 	join_tag = /datum/job/officer
+
+/datum/job/officerjr
+	title = "Junior Marshal Officer"
+	flag = OFFICERJR
+	department = DEPARTMENT_SECURITY
+	department_flag = SECURITY
+	faction = MAP_FACTION
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the Warrant Officer"
+	difficulty = "Hard."
+	selection_color = "#a7bbc6"
+	wage = WAGE_LABOUR_INTERN
+	health_modifier = 10
+	disallow_species = list(FORM_SOTSYNTH, FORM_AGSYNTH, FORM_NASHEF)
+
+	outfit_type = /decl/hierarchy/outfit/job/security/ihoper/jr
+
+	access = list(
+		access_security, access_moebius, access_engine, access_mailsorting, access_eva, access_forensics_lockers, access_medspec,
+		access_sec_doors, access_brig, access_maint_tunnels, access_morgue, access_external_airlocks, access_prospector, access_kitchen, access_medical_suits
+	)
+
+	stat_modifiers = list(
+		STAT_ROB = 25,
+		STAT_TGH = 25,
+		STAT_VIG = 20,
+	)
+
+	perks = list(PERK_CODESPEAK, PERK_CHEM_CONTRABAND)
+
+	software_on_spawn = list(/datum/computer_file/program/digitalwarrant,
+							 /datum/computer_file/program/camera_monitor)
+
+	description = "The Marshal Officer forms the brunt of the Marshals, internally enforcing law and keeping the peace.<br>\
+	You are learning to become a Marshal and are expected to be a learn how to problem solve and handle descalate situations, reach peaceful agreements, and maintain public trust.<br>\
+	Keep your weapons holstered unless the situation demands otherwise - exercise good judgment and follow Blackshield orders.<br>\
```

Comment:
```suggestion
	Keep your weapons holstered unless the situation demands otherwise - exercise good judgment and follow orders from full Marshals.<br>\
```

Reply:
Marshals do not take orders from Blackshield under normal circumstances